synapse/tests/test_lib_modelrev.py

```diff
@@ -428,3 +428,9 @@ async def test_modelrev_0_2_22(self):
 
             for node in nodes:
                 self.eq(node.props.get('type'), 'shared')
+
+    async def test_modelrev_0_2_23(self):
+
+        with self.getLoggerStream('synapse.lib.modelrev') as stream:
```

Comment:
what is stream doing here?

Reply:
This may be vestigial from @MichaelSquires 's edits. I'll remove.

Reply:
( looks like he already removed it when he separated the modelrev PRs )